README.md

```diff
@@ -0,0 +1,67 @@
+# Thonny Flatpak
+
+This is the official [Thonny](https://thonny.org/) Flatpak.
+
+## Users
+
+This section covers installation instructions and additional information for Thonny users.
```

Comment:
no user will see that, so that you know.

Reply:
Hopefully you're right, but I imagine it's possible.